framework/include/outputs/OutputWarehouse.h

```diff
@@ -189,6 +189,43 @@ class OutputWarehouse
    */
   void bufferConsoleOutputsBeforeConstruction(bool buffer) { _buffer_action_console_outputs = buffer; }
 
+public:
+  /**
+   * Calls the initialSetup function for each of the output objects
+   * @see FEProblemBase::initialSetup()
+   */
+  void initialSetup();
```

Comment:
This is a big change. We are moving all of these setup methods from private to public?